subprojects/configuration-cache/src/main/kotlin/org/gradle/configurationcache/metadata/ProjectMetadataController.kt

```diff
@@ -150,10 +154,20 @@ class ProjectMetadataController(
         val dependencies = readDependencies(componentId)
         val variants = readVariants(factory).toSet()
 
+        val dependencyMetadata = factory.create(Describables.of(configurationName, "dependencies"), ValueCalculator {
+            ConfigurationDependencyMetadata(
+                dependencies, emptySet(), emptyList(),
+            )
+        })
+
+        val artifactMetadata = factory.create(Describables.of(configurationName, "artifacts"), ValueCalculator {
+            ImmutableList.of<LocalComponentArtifactMetadata>()
```

Comment:
💅 This is an immutable list with nothing in it, shouldn't this use `emptyList<LocalComponentArtifactMetadata>()`?

Reply:
Oh, I see, it can't because `LocalConfigurationMetadata` returns an `ImmutableList` and this flows to there.